packages/clay-data-provider/stories/index.tsx

```diff
@@ -209,4 +209,42 @@ storiesOf('Components|ClayDataProvider', module)
 	))
 	.add('with variables change and storage', () => (
 		<ClayDataProviderWithVariablesAndStorage />
-	));
+	))
+	.add('toggle poll', () => {
+		const [poll, setPoll] = useState(false);
+
+		const {resource} = useResource({
+			link:
+				'https://api-public.sandbox.pro.coinbase.com/products/BTC-USD/trades',
```

Comment:
Could we maybe point to something else? Bitcoin doesn't always have a great reputation, and might throw some people off to see a link to coinbase.

Reply:
Oh yeah, np. Can i change for another 😁